src/systems/ec/implementations/react/templates/compositions/navigation-lists/examples/Default.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Link from '@ecl/ec-react-component-link';
+import { UnorderedList, UnorderedListItem } from '@ecl/ec-react-component-list';
+
+export default () => (
+  <div>
```

Comment:
I'm not a big fan of this "naked" div here... Though I don't see any other semanic tag that may fit.

Reply:
Indeed, it's ugly. It will make more sense when used in a composition of composition, when you need to apply classes on the container (the `div` here). For now, I don't see any better solution...